cropper/src/main/java/com/canhub/cropper/CropImageActivity.kt

```diff
@@ -61,15 +61,45 @@ open class CropImageActivity :
 
         if (savedInstanceState == null) {
             if (cropImageUri == null || cropImageUri == Uri.EMPTY) {
-                when {
-                    cropImageOptions.imageSourceIncludeGallery &&
+                if (cropImageOptions.showIntentChooser) {
+                    val ciIntentChooser = CropImageIntentChooser(
+                        activity = this,
+                        callback = object : CropImageIntentChooser.ResultCallback {
+                            override fun onSuccess(uri: Uri?) {
+                                onPickImageResult(uri)
+                            }
+
+                            override fun onCancelled() {
+                                setResultCancel()
+                            }
+                        }
+                    )
+                    cropImageOptions.apply {
```

Comment:
I used `with (cropImageOptions) { }` earlier but ktlint didn't accept it. Should I use `let`?

Reply:
I had used `with (cropImageOptions) { ... }` before solely to avoid repetition of `cropImageOptions` but ktlint didn't accept it. Would `let` be better instead?

Reply:
yeah